tests/parsers/thermo_fisher_nanodrop_eight/testdata/Thermo_NanoDrop_Eight_example01.json

```diff
@@ -7,7 +7,7 @@
                     "measurement time": "2020-05-26T15:36:00+00:00",
                     "measurement document": [
                         {
-                            "measurement identifier": "THERMO_FISHER_NANODROP_EIGHT_TEST_ID_7",
+                            "measurement identifier": "THERMO_FISHER_NANODROP_EIGHT_TEST_ID_0",
```

Comment:
Note to reviewer: the diffs in ids here are because we now only generate a uuid when we're going to make a measurement document, whereas previously we generated uuids for every row in the dataframe, and then threw some out (note how the test ids now start at 0)